### PR TITLE
delete base token config

### PR DESCRIPTION
**Pull request by pfi79**

### CODE REVIEW

File: token/get_fee_transfer.go [hunk 1]
```diff
@@ -34,15 +34,16 @@ type FeeTransferResponseDTO struct {
 }
 
 func (bt *BaseToken) QueryGetFeeTransfer(req FeeTransferRequestDTO) (*FeeTransferResponseDTO, error) {
-	if err := bt.loadConfigUnlessLoaded(); err != nil {
+	cfg, err := bt.loadConfigUnlessLoaded()
```
Comment (muzykantov) [inline token/get_fee_transfer.go:37]: Maybe just loadConfig?

Comment (pfi79) [reply token/get_fee_transfer.go:37]: rename the method?

**Review by muzykantov — COMMENTED** [review]
Everything is fine except naming. I'd recommend to rename the function more relevant coz behavior has been changed.

File: token/get_fee_transfer.go (hunk 1 above)
Comment (pfi79) [reply token/get_fee_transfer.go]: fix

**Review by muzykantov — APPROVED** [review]

**Pull request merged**